test/services/notify/notify-email.service.test.js

```diff
@@ -0,0 +1,159 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, afterEach, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const { NotifyClient } = require('notifications-node-client')
+const { notifyTemplates } = require('../../../app/lib/notify-templates.lib.js')
+
+// Thing under test
+const NotifyEmailService = require('../../../app/services/notify/notify-email.service.js')
+
+describe('Notify - Email service', () => {
+  const stubNotify = process.env.STUB_NOTIFY || true // Used to perform integration tests with notify
```

Comment:
I'd move this to `config/notify.config.js` so the config can all be put in one place (see `config/log.config.js` for an example of where we have previously put 'test-only' config).

Making this change also means _all_ config comes from the `config/` files, and folks only have to look in there to see what can be changed. We aim not to have any 'hidden' config values elsewhere because of direct references to `process.env`.